jenkins/bootstrap_test.py

```diff
@@ -2296,6 +2296,8 @@ def test_valid_job_envs(self):
                     if (
                             '--env-file=jobs/platform/gce.env' not in args
                             and '--env-file=jobs/platform/gke.env' not in args
+                            and '--env-file=jobs/platform/gke-staging.env' not in args
+                            and '--env-file=jobs/platform/gke-prod.env' not in args
```

Comment:
gke jobs still references to `jobs/platform/gke.env` so those two might not needed here.